### PR TITLE
Thick stripes

### PR DESCRIPTION
**Pull request by stefnotch**
image

### CODE REVIEW

**Comment by phcreery** [comment]
There are a bunch of other commits in this that are unrelated, can you pull or whatever is needed ahah, IDK the correct git terminology for that.

File: src/ui/QuantumDocument.vue [hunk 1]
```diff
@@ -300,9 +300,13 @@ export default defineComponent({
...
   --selected-background-color: rgba(68, 148, 202, 0.24);
   --selected-color: rgba(57, 131, 180, 0.459);
-  background-size: var(--grid-cell-size-x) var(--grid-cell-size-y);
+
+  background-size: var(--grid-cell-size-x) var(--grid-cell-size-y), var(--grid-cell-size-x) var(--grid-cell-size-y),
+    calc(var(--grid-cell-size-x) * 5) calc(var(--grid-cell-size-y) * 5), calc(var(--grid-cell-size-x) * 5) calc(var(--grid-cell-size-y) * 5);
   background-image: linear-gradient(to right, var(--grid-color) 1px, transparent 1px),
-    linear-gradient(to bottom, var(--grid-color) 1px, transparent 1px);
+    linear-gradient(to bottom, var(--grid-color) 1px, transparent 1px), linear-gradient(to right, rgba(71, 162, 223, 0.7) 1px, transparent 1px),
+    linear-gradient(to bottom, rgba(71, 162, 223, 0.7) 1px, transparent 1px);
```
Comment (phcreery) [inline src/ui/QuantumDocument.vue]: For the `rgba()` values, can you swap them for `var(--grid-color)`? This also makes the 'bold' lines dynamic, a little more subtle, and easier to see.

**Comment by stefnotch** [comment]
> 
> 
> There are a bunch of other commits in this that are unrelated, can you pull or whatever is needed ahah, IDK the correct git terminology for that.

That's odd, the commits were still showing up despite already having been merged into the antd_UI branch. I fixed that by locally deleting the branch, creating a new branch called `thick-stripes`, doing the changes again and finally, force pushing.

File: src/ui/QuantumDocument.vue (hunk 1 above)
Comment (stefnotch) [reply src/ui/QuantumDocument.vue]: I tried it out, this does look much nicer indeed

**Comment by phcreery** [comment]
Okay, It looks good! I'll merge

**Pull request merged**